### PR TITLE
css-two-column-layout - Alex Gonzalez

### PR DESCRIPTION
**Pull request by gonzalez-alex**
Updated SS:

css-two-column-layout

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      </div>
+      <div class="column-fifth">
+        <button>LEARN MORE</button>
+      </div>
+      <div class="column-fifth">
+      </div>
+      <div class="column-fifth">
+      </div>
+    </div>
+  </div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: You should never have empty columns in a layout class system to create space. Instead, make a class that you can apply to the `row` that utilizes `justify-content` to place the single `column` with content in the center.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-two-column-layout/index.html (hunk 1 above)
Comment (gonzalez-alex) [reply css-two-column-layout/index.html]: ah ic, that makes things much simpler.. haha def overthought that one..

**Comment by gonzalez-alex** [comment]
css-two-column-layout

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**